### DISCUSSION TITLE
HW2 about accuracy

### DISCUSSION THREAD

**Discussion by flrina0305** [seominjoon/kaist-ai605#36] [Q&A] [answered]
Hi, professor! 
I have a question about goal accuracy for assignment2. 
How much is goal accuracy for problem2 and 3 model approximately?

Thanks in advance :)

**Comment by seominjoon** [comment, marked as answer]
It seems many people had difficulty getting good accuracy because the assignment instruction was not comprehensive enough. Don't worry too much about the accuracy for this assignment. We will rather grade on whether your implementation is in the right direction.
